d2bs/kolbot/libs/common/Town.js

```diff
@@ -77,6 +77,8 @@ var Town = {
 		Attack.weaponSwitch(Attack.getPrimarySlot());
 
 		this.heal();
+		Mercenary.revive();
+		Mercenary.hire(Config.UseMerc);
```

Comment:
The actions are done in this order for compatibility with the Mercenary autoEquip pending pull request.  When the bot is shopping/identifying items it might want to keep items for equipping the merc, and the merc will need to be present for this to work properly.

Hire and revive are separated because they are discrete actions.